ql/termstructures/yield/nonlinearfittingmethods.hpp

```diff
@@ -52,18 +59,30 @@ namespace QuantLib {
                                       ext::shared_ptr<OptimizationMethod>(),
                                   const Array& l2 = Array(),
                                   Real minCutoffTime = 0.0,
-                                  Real maxCutoffTime = QL_MAX_REAL);
+                                  Real maxCutoffTime = QL_MAX_REAL,
+                                  Size numCoeffs = 9,
+                                  Real fixedKappa = QL_NULL_REAL);
```

Comment:
`QL_NULL_REAL` is a macro we're using to implement nulls, but the idea is to keep it private to the definition of the `Null` class.  Here I'd use `Null<Real>()` instead.

Reply:
Happy to go with the construct you think is best.